src/widgets/colorchooser.h

```diff
@@ -0,0 +1,251 @@
+#ifndef COLORCHOOSER_H
+#define COLORCHOOSER_H
+
+#include <QLineEdit>
+#include <QProxyStyle>
+#include <QWidget>
+#include <qapplication.h>
+#include <qgroupbox.h>
+#include <qlabel.h>
+#include <qpushbutton.h>
+#include <qslider.h>
+#include <qstackedwidget.h>
+#include <qdebug.h>
+#include <qpainter.h>
+#include <QSpinBox>
+#include <QDesktopWidget>
+
+
+
+class ColorCircle;
+class CustomStyle1 : public QProxyStyle
+{
+
+public:
+
+	using QProxyStyle::QProxyStyle;
+
+	int styleHint(QStyle::StyleHint hint,
+
+		const QStyleOption* option = 0,
+
+		const QWidget* widget = 0,
+
+		QStyleHintReturn* returnData = 0) const
+	{
+		if (hint == QStyle::SH_Slider_AbsoluteSetButtons) 	return (Qt::LeftButton | Qt::MidButton | Qt::RightButton);
+
+		return QProxyStyle::styleHint(hint, option, widget, returnData);
+	}
+};
+
+
+
+class CustomBackground : public QWidget
+{
+	Q_OBJECT
+public:
+	bool isBig = false;
```

Comment:
What is `isBig`, use a more descriptive name indicative of what this actually does